src/main/java/com/cb/bl/ShopBL.java

```diff
@@ -0,0 +1,44 @@
+package com.cb.bl;
+
+public class ShopBL {
+
+    private int id;
+    private String name;
+    private String target;
+    private int power;
+
+    public ShopBL() {
+    }
+
```

Comment:
Maybe it's not necessary to use  empty contructor if there is no more constructors?

Reply:
I suggest to leave it, to conform to Beans standard..

Reply:
We are not using here, because if there is no more constructors, it's redundant to declare empty constructor. It's not a mistake, just a good practice.

Reply:
I was following different convention that i read somewhere. Okay, deleted it